src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -53,6 +57,8 @@ public AppealController(AppealService appealService) {
                                                @PathVariable("company-id") final String companyId,
                                                @Valid @RequestBody final Appeal appeal) {
 
+        appeal.setCreatedAt(LocalDateTime.now());
```

Comment:
Are we moving this back into the service as it's not really needed here now?

Reply:
Yeah that would be preferable